packages/web/src/pages/landing-page/components/Description.tsx

```diff
@@ -165,7 +165,9 @@ const Description = (props: DescriptionProps) => {
           <animated.div
             className={styles.textDescription}
             style={{
+              // @ts-ignore
```

Comment:
Are these difficult to type?

Reply:
yeah i think i just don't understand how it's possible to type them. the style object there is the CSSProperties type that comes from react, so `transform` type is string, but this transform type is `react-springs` version, which is compatible with animated.div, but weirdly not typed differently from react. maybe old version of spring? will look more into this though

Reply:
interestingly if you look at that file, there are TONS of ts-ignore, guessing something is up with react-spring, added a ticket to address this in general, since there are quite a few cases in many files